Mixpanel/app_extension_module.modulemap

```diff
@@ -1,5 +1,5 @@
 framework module MixpanelAppExtension {
-    umbrella header "Mixpanel.h"
+    header "Mixpanel.h"
```

Comment:
i'm guessing this change is the same? I noticed MixpanelPeople.h is imported in Mixpanel.h. Is there another reason why you decided to not use the `umbrella` anymore?